src/pe/import.rs

```diff
@@ -129,7 +129,7 @@ impl<'a> SyntheticImportLookupTableEntry<'a> {
                     use self::SyntheticImportLookupTableEntry::*;
                     if bitfield.is_ordinal() {
                         let ordinal = bitfield.to_ordinal();
-                        debug!("importing by ordinal {:#x}", ordinal);
+                        debug!("importing by ordinal {:#x} ({})", ordinal, ordinal);
```

Comment:
if you don't mind we should start porting this to `{ordinal:#x} ({ordinal})` whenever we encounter older style logs

Reply:
you have a lot of PRs up, I don't think this should waste your time fixing it; can do in another pass